sanic/sanic.py

```diff
@@ -176,6 +177,15 @@ def converted_response_type(self, response):
         :return: Nothing
         """
         try:
+            request_started.send(request)
```

Comment:
Blinker says it has been optimized for that case. (That doesn't replace a good measurement, of course :) )